cypress/e2e/shared/checks.test.ts

```diff
@@ -226,17 +226,25 @@ describe('Checks', () => {
   })
 
   it('can create and filter checks', () => {
-    cy.get<Organization>('@org').then((org: Organization) => {
-      cy.intercept('POST', `/api/v2/query?orgID=${org.id}`, req => {
-        if (req.body.query.includes('_measurement')) {
-          req.alias = 'measurementQuery'
+    cy.get<Organization>('@org').then(({id: orgID}: Organization) => {
+      cy.intercept(
+        'POST',
+        `/api/v2/query?${new URLSearchParams({orgID})}`,
```

Comment:
is the main change here the `new URLSearchParams({orgID})` rather than `?orgID=${orgID}` ?

Reply:
yes, because it handles the URL encoding for the iox orgIDs. Which were errorring otherwise.